app.py

```diff
@@ -22,8 +23,17 @@ def get_all():
     except Exception as e:
         return(str(e))
 
+@app.route('/api/v1/news', methods = ['GET'])
+def get_api():
+    try: 
+        response = requests.get('https://spaceflightnewsapi.net/api/v1/articles')
+        return response.json()
+    except Exception as response:
+        return(str("Bad Request"))
+
```

Comment:
Added in the api functionality for space news. It will error out and return "Bad Request" if something goes wrong.